src/h3lib/include/sphereCapTables.h

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2016-2018, 2020-2021 Uber Technologies, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/** @file sphereCapTables.h
+ *  @brief Shared lookup tables for SphereCap calculations.
+ *
+ *  These values are used by the library implementation and tests. They are
+ *  defined in a header as `static const` arrays so that including translation
+ *  units receive private copies without exporting additional symbols.
+ */
+
+#ifndef SPHERE_CAP_TABLES_H
+#define SPHERE_CAP_TABLES_H
+
+#include "constants.h"
+
+#define H3_SPHERE_CAP_SCALE_FACTOR ((double)1.01)
```

Comment:
This is for including some slight overage in the test radius to ensure cells are marked as being included? I assume it was empirically chosen?